app/uk/gov/hmrc/tariffclassificationfrontend/controllers/RejectCaseController.scala

```diff
@@ -70,6 +57,33 @@ class RejectCaseController @Inject()(verify: RequestActions,
       renderView(c => c.status == REJECTED, c => successful(views.html.confirm_rejected(c)))
     }
 
+  def postRejectCase(reference: String): Action[MultipartFormData[Files.TemporaryFile]] = (verify.authenticated andThen verify.casePermissions(reference) andThen
```

Comment:
I managed to make this a less duplicate code-y in https://github.com/hmrc/tariff-classification-frontend/pull/219/files#diff-e8718752c65054fc302e1d52c166ee48R70

Reply:
I pulled your change in - looks better